src/shipstats.c

```diff
@@ -149,36 +154,36 @@ static const ShipStatsLookup ss_lookup[] = {
    DI_ELEM( SS_TYPE_D_JUMP_WARMUP,        jump_warmup,         N_("Jump Warmup") ),
    D__ELEM( SS_TYPE_D_MINING_BONUS,       mining_bonus,        N_("Mining Bonus") ),
 
-   A__ELEM( SS_TYPE_A_THRUST,             thrust,              N_("kN/tonne Thrust") ),
-   A__ELEM( SS_TYPE_A_TURN,               turn,                N_("deg/s Turn Rate") ),
-   A__ELEM( SS_TYPE_A_SPEED,              speed,               N_("m/s Maximum Speed") ),
-   A__ELEM( SS_TYPE_A_ENERGY,             energy,              N_("MJ Energy Capacity") ),
-   A__ELEM( SS_TYPE_A_ENERGY_REGEN,       energy_regen,        N_("MW Energy Regeneration") ),
-   AI_ELEM( SS_TYPE_A_ENERGY_REGEN_MALUS, energy_regen_malus,  N_("MW Energy Usage") ),
-   AI_ELEM( SS_TYPE_A_ENERGY_LOSS,        energy_loss,         N_("MW Energy Usage") ),
-   A__ELEM( SS_TYPE_A_SHIELD,             shield,              N_("MJ Shield Capacity") ),
-   A__ELEM( SS_TYPE_A_SHIELD_REGEN,       shield_regen,        N_("MW Shield Regeneration") ),
-   AI_ELEM( SS_TYPE_A_SHIELD_REGEN_MALUS, shield_regen_malus,  N_("MW Shield Usage") ),
-   A__ELEM( SS_TYPE_A_ARMOUR,             armour,              N_("MJ Armour") ),
-   A__ELEM( SS_TYPE_A_ARMOUR_REGEN,       armour_regen,        N_("MW Armour Regeneration") ),
-   AI_ELEM( SS_TYPE_A_ARMOUR_REGEN_MALUS, armour_regen_malus,  N_("MW Armour Damage") ),
-   A__ELEM( SS_TYPE_A_DAMAGE,             damage,              N_("MW Damage") ),
-   A__ELEM( SS_TYPE_A_DISABLE,            disable,             N_("MW Disable") ),
-
-   A__ELEM( SS_TYPE_A_CPU_MAX,            cpu_max,             N_("CPU Capacity") ),
-   A__ELEM( SS_TYPE_A_ENGINE_LIMIT,       engine_limit,        N_("Engine Mass Limit") ),
-   A__ELEM( SS_TYPE_A_FUEL_REGEN,         fuel_regen,          N_("Fuel Regeneration") ),
-   A__ELEM( SS_TYPE_A_ASTEROID_SCAN,      asteroid_scan,       N_("Asteroid Scanner Range") ),
-   A__ELEM( SS_TYPE_A_NEBULA_VISIBILITY,  nebu_visibility,     N_("Nebula Visibility") ),
+   A__ELEM( SS_TYPE_A_THRUST,             thrust,              N_("Thrust"),              N_("t") ),
+   A__ELEM( SS_TYPE_A_TURN,               turn,                N_("Turn Rate"),           N_("deg/sec") ),
+   A__ELEM( SS_TYPE_A_SPEED,              speed,               N_("Maximum Speed"),       N_("m/s") ),
```

Comment:
Might make sense to have the unit stuff be set up as const strings instead of repeating.